h2o-core/src/main/java/water/H2O.java

```diff
@@ -1473,7 +1480,7 @@ public static void registerResourceRoot(File f) {
   /** Start the web service; disallow future URL registration.
    *  Blocks until the server is up.  */
   static public void finalizeRegistration() {
-    if (_doneRequests) return;
+    if (_doneRequests || H2O.ARGS.disable_web) return;
     _doneRequests = true;
 
     water.api.SchemaServer.registerAllSchemasIfNecessary();
```

Comment:
Minor: This is a problem for model info serialization introduced by @navdeep-G . He is using schema reflection lookup for find right schema for a model.